src/components/IconToggle/IconToggle.tsx

```diff
@@ -50,12 +61,12 @@ export class IconToggle extends React.Component<IconToggleProps> {
   };
 
   render() {
-    const { labelPlacement, onChange, checked, disabled, ...rest } = this.props;
+    const { labelPlacement, onChange, checked, disabled, animation, ...rest } = this.props;
 
     const content = this._getContent();
-
+    
     return (
-      <span {...styles('root', { checked, disabled, labelPlacement }, rest)}>
+      <span {...styles('root', { checked, disabled, labelPlacement, animation }, rest)} onMouseLeave={this._handleHoverOff}>
```

Comment:
@bamai 
Remind me...
We said it might be possible to remove the component's state, right?
And just use the prop for the animation.
(I mentioned `animation-direction`).
What did we decide about this?

Reply:
I think we decided to remove the state (the last commit is supposed to work without using internal state)

Reply:
Oh, ok, so I guess we can remove the `onMouseLeave` and the `State` interface